File: IntroPC/Tarea 6/Combinatoria/rutinas.cpp
double factorial(int n)
{
  int i;
  double factorial;

  factorial=1;

  for(i=n;i>=1;i=i-1)
    {
      factorial=factorial*i;
    }
  
  return factorial;
}
double combinatoria(double L,double P)
{
  double aux,fac1,fac2;
  fac1=factorial(L);
  fac2=factorial((L-P));
  aux=fac1/fac2;
  return aux;
}

File: IntroPC/Tarea 6/Combinatoria/rutinas.py
#! /usr/bin/env python

def factorial(n):
     factorial=1
     i=n
     while i>=1:
          factorial=factorial*i
          i-=1
     return factorial

def combinatoria(L,P):

     fac1=factorial(L)
     fac2=factorial((L-P))
     aux=fac1/fac2
     return aux
